bitnami/consul/Chart.yaml

```diff
@@ -3,24 +3,26 @@ annotations:
 apiVersion: v2
 appVersion: 1.10.1
 dependencies:
-  - name: common
-    repository: https://charts.bitnami.com/bitnami
-    tags:
-      - bitnami-common
-    version: 1.x.x
-description: Highly available and distributed service discovery and key-value store designed with support for the modern data center to make distributed systems and configuration easy.
+- name: common
```

Comment:
It seems that you applied an auto-formatter incompatible with our yaml conventions. Could you fix it?